src/client/helpers/metadata.js

```diff
@@ -1,75 +1,85 @@
 import omit from 'lodash/omit';
 import SteemConnect from '../steemConnectAPI';
 
-const getMetadata = () => SteemConnect.me().then(resp => resp.user_metadata);
+// sc2 metadata support discontinuation preparation
+// const getMetadata = () => SteemConnect.me().then(resp => resp.user_metadata);
+export const getMetadata = () =>
+  SteemConnect.me().then(resp => {
```

Comment:
Request to steemconnect API should only be sent if there isn't localStorage with user metadata available.

Reply:
great point!

Reply:
done!